packages/components/src/internal/components/notifications/ReleaseNote.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+
+import { getServerContext } from '@labkey/api';
+
+import { withRouter, WithRouterProps } from 'react-router';
+
+import { HelpLink } from '../../util/helpLinks';
+import { biologicsIsPrimaryApp, getPrimaryAppProperties } from '../../app/utils';
+
+export const DISMISSED_STORAGE_PREFIX = '__release_notes_dismissed__';
+
+export const ReleaseNoteImpl: FC<WithRouterProps> = props => {
+    const { location } = props;
```

Comment:
This will need to get changed for the react-router refactor, right @labkey-alan?